Lib/site.py

```diff
@@ -390,6 +390,54 @@ def setcopyright():
         files, dirs)
 
 
+def sethelper():
+    builtins.help = _sitebuiltins._Helper()
+
+def _register_readline():
```

Comment:
While it isn't documented, the lack of an underscore prefix on `site.enablerlcomplete` means we need to treat it as an implicitly public API.

Reply:
I've replaced my `_register_readline` with the original `enablerlcomplete`